src/schema.js

```diff
@@ -30,20 +30,20 @@ const Category = new GraphQLEnumType({
 const Author = new GraphQLObjectType({
   name: "Author",
   description: "Represent the type of an author of a blog post or a comment",
-  fields: () => ({
+  fields: {
```

Comment:
Here we use an arrow function for the lazy evaluation. I think we need to have it.  Could you do it for other places as well. (only when defining fields)